### ISSUE TITLE
Item_Links library breaks ownership (or related)

### ISSUE THREAD

**Issue by ImaBuggyMess** [ThePix/QuestJS#76]
Title.

settings.libraries.push('item_links')  breaks item ownership
[settings.txt](https://github.com/ThePix/QuestJS/files/8226938/settings.txt)
[data.txt](https://github.com/ThePix/QuestJS/files/8226939/data.txt)

**Uses most recent files, not Quest 0.9 zip**

**Comment by ThePix** [comment]
I have to admit that I do not like links in the text so never use them in test games, which is why bugs can occur, so thanks for spotting.

I have uploaded two fixes.

One issue was Quest was trying to use `player` before it was set, and that could be related to ownership, as I had to do something similar. It now just bails if player is not set.

The other issue is that the logic for when links are disabled was very strange. I do not get how it was working before, and I maybe missing something, so try it out and see how it goes. I will leave this open for a while.
